docker/docker-compose.yml

```diff
@@ -49,23 +49,22 @@ services:
       - "redis"
       - "grafana"
       - "tempo"
-    image: wasmcloud/wasmcloud_host:latest
+    image: wasmcloud/wasmcloud:0.79.0
```

Comment:
I think we decided here previously to keep it as `latest` so we don't accidentally forget about this docker compose forever. Since you're using wadm:latest I think this should be `wasmcloud/wasmcloud:latest`